Update trigger-on-a-new-artifact.md (#1710)
harness/developer-hub: docs/platform/11_Triggers/trigger-on-a-new-artifact.md

### Important notes

> * If more than one artifact is collected during the polling interval (one minute), only one deployment will be started and will use the last artifact collected.

Reviewer: ```suggestion
* If more than one artifact is collected during the polling interval (one minute), only one deployment starts using the latest artifact.
```

* The trigger is executed based on **file names** and not metadata changes.
* Do not trigger on the **latest** tag of an artifact, such as a Docker image. With latest, Harness only has metadata, such as the tag name, which has not changed, and so Harness does not know if anything has changed. The trigger will not be executed.
* In Harness, you can select who is able to create and use triggers within Harness, but you must use your repository's RBAC to control who can add the artifacts or initiate the events that start the Harness trigger.

> * Harness recommends that you submit a tag, or push an artifact when you create a trigger for the first time to verify whether the flow is working properly. This ensures that the trigger will execute and the pipeline will run as expected when subsequent tags are pushed.\
> * When a trigger that is enabled is toggled to disabled, the polling stops. Upon renabling the trigger the artifact polling restarts. Note that this is treated as a new polling job and Harness recommends that you submit a tag or push an artifact and verify the flow.

Reviewer: ```suggestion
* Whenever you create a trigger for the first time, Harness recommends submitting a tag or pushing an artifact to verify its functionality. By doing this, the trigger will execute and the pipeline will run as expected when subsequent tags are pushed.
* The polling stops when you disable a trigger. Artifact polling restarts after reenabling the trigger. Harness recommends that you submit a tag or push an artifact and verify the flow as this is treated as a new polling job.
```
